app/src/test/kotlin/fr/free/nrw/commons/utils/ImageUtilsTest.kt

```diff
@@ -0,0 +1,15 @@
+package fr.free.nrw.commons.utils
+
+import org.junit.Assert.assertEquals
+import org.junit.Test
+
+class AlgoCheck {
```

Comment:
File should have same name as primary class, so class should be called ImageUtilsTest